builds/flu/flu_titers.py

```diff
@@ -48,10 +49,73 @@ def HI_model(process):
         node.attr["cTiterSub"] = node.cTiterSub
         node.attr["dTiterSub"] = node.dTiterSub
 
+def vaccine_distance(process, attributes=['ep', 'dTiter', 'dTiterSub']):
```

Comment:
This function is very clear to read and will be generally useful for future titer work. To make the function more modular, I would replace the `process` input with inputs for a `tree` and a list of `vaccine_strains` and have the function return the `distance_to_vaccine` list as output. This will allow anyone to use this function independently from a `process` instance (e.g., in a Jupyter notebook).